manifests/t/trieck/comexplore/1.0/trieck.comexplore.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: trieck.comexplore
```

```suggestion
PackageIdentifier: PolarGoose.comexplore
```
This should match the GitHub Repo to identify the source of the release, especially since a folder already exists for `PolarGoose`